ipatests/test_integration/test_automember.py

```diff
@@ -0,0 +1,316 @@
+#
+# Copyright (C) 2019  FreeIPA Contributors see COPYING for license
+#
+"""This covers tests for automember feature
+"""
+from __future__ import absolute_import
+
+import textwrap
+
+from tempfile import NamedTemporaryFile
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestAutomember(IntegrationTest):
+    """
+    Tests of Automember feature.
+    Covers unmembering feature described in
+    https://pagure.io/389-ds-base/issue/50077
+    """
+    topology = 'line'
+
+    def test_modify_user_entry_unmembership(self):
+        # https://pagure.io/389-ds-base/issue/50077
+        tasks.kinit_admin(self.master)
+        master = self.master
+        search_cmd = [
+            'ldapsearch', '-x',
+            '-D', 'cn=directory manager',
+            '-w', master.config.dirman_password,
+            '-b', 'cn=Auto Membership Plugin,cn=plugins,cn=config']
```

Comment:
I would add `'-s', 'base'` option to make a base search (only look for the entry `cn=Auto Membership Plugin,cn=plugins,cn=config` and not for subordinates).